charts/stable/external-dns/Chart.yaml

```diff
@@ -38,5 +38,5 @@ sources:
   - https://github.com/kubernetes-sigs/external-dns
   - https://github.com/trueforge-org/truecharts/tree/master/charts/stable/external-dns
 type: application
-version: 7.12.3
+version: 7.12.4
```

Comment:
@alfi0812 This should never have been approved, this had to be a feature not a patch release.